repository/src/main/java/org/lfenergy/compas/scl/data/model/SclType.java

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2021 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.scl.data.model;
+
+public enum SclType {
```

Comment:
Maybe adding Todo/Story for adding all the SCL types?
Or not doing it and assuming it will be done in the future?